models/classes/resources/Service/ClassCopier.php

```diff
@@ -42,17 +42,22 @@ class ClassCopier implements ClassCopierInterface
     /** @var InstanceCopierInterface */
     private $instanceCopier;
 
+    /** @var ClassMetadataMapper */
+    private $classMetadataMapper;
+
     /** @var string[] */
     private $copiedClasses = [];
 
     public function __construct(
         RootClassesListServiceInterface $rootClassesListService,
         ClassMetadataCopierInterface $classMetadataCopier,
-        InstanceCopierInterface $instanceCopier
+        InstanceCopierInterface $instanceCopier,
+        ClassMetadataMapper $classMetadataMapper
```

Comment:
Use the interface, if possible :) 

```suggestion
        ClassMetadataMapperInterface $classMetadataMapper
```